specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/SensitivityLabels.json

```diff
@@ -33,6 +33,18 @@
           {
             "$ref": "#/parameters/DatabaseNameParameter"
           },
+          {
```

Comment:
Can I know why add this feature in a old version? suggest to support in a new API version as it will be a breaking change in a existing version. and also you may need add a description for the parameter

Reply:
@akning-ms
Hi, this feature was added a while ago, we just noticed that the swagger is not updated and does not reflect that.

BTW, why is it a breaking change?
As far as I know, the API can run with or without these parameters, and no parameters are being removed.

Reply:
Thanks, when this feature added. seems your missed properties is already in new added version 2020-08-01-preview. so it is not required to change a old version like 2017-03-01-preivew and 2018-06-01-preview. and also V5 SDK will base on 2020-08-01-preivewe version according to @ericshape.  
As for why it is a breaking change. you can refer to policy http://aka.ms/bcforapi section 5: examples it is suggest to add it in new version like 2020-08-01-privew

Reply:
@bashahee 
1. can you verify whether the json files in https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/ have contained your changes? I just update this folder in early December. 
2. can you use the latest 2020-08-01 json file to replace this line in V4 SDK: https://github.com/Azure/azure-rest-api-specs/blame/master/specification/sql/resource-manager/readme.md#L86 ?

Reply:
@ericshape
1. Yes.
2. Yes, it will be done in a different PR, this PR will be closed.